### ISSUE TITLE
[Feature Request]: Ability to sort variables/columns

### ISSUE THREAD

**Issue by LisaD84** [jasp-stats/jasp-issues#2786]
### Description

Possibility to manually resort variables

### Purpose

Faster analysis

### Use-case

all files with many variables

### Is your feature request related to a problem?

It's annoying to have to cut and paste the variables rather than to shift them

### Describe the solution you would like

Be able to grab and pull, ideally multiple variables at once

**Comment by boutinb** [comment]
Hi @LisaD84 , I'm not sure what you mean here. You can already sort the variables by name or by type by clicking the sort icon at the top-left of the Variables list:
![Screenshot 2024-06-19 at 09 40 07](https://github.com/jasp-stats/jasp-issues/assets/503332/37ad4187-1639-4153-b994-406d18ac9b03)
Also you can easily select several variables, and drag-and-drop them together.

**Comment by tomtomme** [comment]
@boutinb 
This is about the data viewer. Currently you have to select and then cut/copy and paste, like:
![grafik](https://github.com/jasp-stats/jasp-issues/assets/4428185/984c2fe2-02db-4135-8f2f-e087224f85e5)

@LisaD84 wants this via drag and drop, like in excel. Correct?

**Comment by LisaD84** [comment]
Yes, exactly, it would be really helpful to be able to drag and drop. I often find it necessary to reorganize variables to make analyses steps faster.

Von: Thomas Langkamp ***@***.***>
Gesendet: Mittwoch, 19. Juni 2024 15:34
An: jasp-stats/jasp-issues ***@***.***>
Cc: Diamond Lisa ***@***.***>; Mention ***@***.***>
Betreff: Re: [jasp-stats/jasp-issues] [Feature Request]: Ability to sort variables (Issue #2786)

@boutinb This is about the data viewer. Currently you have to select and then cut/copy and paste, like: grafik. png (view on web) @LisaD84 wants this via drag and drop, like in excel. Correct? — Reply to this email directly, view it on


@boutinb<https://github.com/boutinb>
This is about the data viewer. Currently you have to select and then cut/copy and paste, like:
grafik.png (view on web)<https://github.com/jasp-stats/jasp-issues/assets/4428185/984c2fe2-02db-4135-8f2f-e087224f85e5>

@LisaD84<https://github.com/LisaD84> wants this via drag and drop, like in excel. Correct?

**Comment by tomtomme** [comment]
@LisaD84 
note there is work in progress now on something very related to this: https://github.com/jasp-stats/jasp-issues/issues/3424#issuecomment-3586403575
